nni/tools/jupyter_extension/management.py

```diff
@@ -1,11 +1,14 @@
 import json
 from pathlib import Path
 import shutil
+import os
 
 from jupyter_core.paths import jupyter_config_dir, jupyter_data_dir
 
 import nni_node
 
+jupyter_lab_version = os.popen('jupyter lab --version').read().rstrip('\n').split('.')[0]
```

Comment:
`jupyterlab.__version__` should be more reliable and elegant.

Reply:
Thanks， changed